### DISCUSSION TITLE
UI removed?

### DISCUSSION THREAD

**Discussion by scriptsengineer** [MenacingMecha/godot-psx-style-demo#4] [Q&A] [answered]
Has the UI that existed in the Demo been removed for godot 4.0?

**Comment by MenacingMecha** [comment, marked as answer]
Yes, all the breaking changes in 4.0 meant updating the demo UI would take a lot more work than just updating the shaders, on top of features being removed, so is shelved until both vertex lighting and the 3.0-style fog are in 4.0 again.
